core/src/main/java/edu/wpi/grip/core/operations/Operations.java

```diff
@@ -30,5 +32,9 @@ public static void addOperations(EventBus eventBus) {
         eventBus.post(new OperationAddedEvent(new NewPointOperation()));
         eventBus.post(new OperationAddedEvent(new NewSizeOperation()));
         eventBus.post(new OperationAddedEvent(new MatFieldAccessor()));
+
+        Operation publishContours = new NTPublishOperation<>(ContoursReport.class);
+        eventBus.register(publishContours);
```

Comment:
Oh, okay. Not really a fan of registering operations on the event bus but sure, whatever.